### PR TITLE
Group photo - sample for frontpage static image

### PR DESCRIPTION
**Pull request by fbbunny**
In order to make a design such as on this picture, this would be a sample background picture untill we make a better one. 
track

### CODE REVIEW

File: static/css/style.css [hunk 1]
```diff
@@ -132,16 +152,17 @@ strong{
 }
 
 #frontpage{
-	height: 300px;
+	height: 400px;
 	padding: 5px 5px 20px 5px;
+	background: url(groupmeet.jpg) no-repeat fixed;
```
Comment (livike) [inline static/css/style.css]: By running the page, the image is not showing for me...

Comment (goranche) [inline static/css/style.css]: are you sure you're running her branch? can you manually check if the image exists on your computer

File: static/css/style.css [hunk 2]
```diff
@@ -35,6 +37,18 @@ footer {
 	text-align: auto;
 }
 
+#address {
+	color: #FFFFFF
+}
+.container-fluid {
+	background-color: #734F43;
+}
+.navbar {
```
Comment (livike) [inline static/css/style.css]: To make a navigation bar fixed, you can add to the html nav tag add the "navbar-fixed-top" class. and  delete this .navbar definition from the style.css

File: static/css/style.css [hunk 3]
```diff
@@ -51,13 +65,13 @@ footer {
...
-	padding: 10px 0px 5px 990px; 
+float:left;
+
 }
 h1 {
 	width: 350px;
-	text-align: center;
 	position: relative; 
-	margin:-100px 0 0 -150px;
+	margin: 45px 0px 105px -150px;
```
Comment (livike) [inline static/css/style.css]: If the fixed navbar overlapping the content (usually the h1 title), you can simply add to the container class a margin-top: 60px. Yo don't need to modify the h1,h2,h3 elements styling or to define a new one with ID as you did.
Please remove the h1, h2, h3 modifications from the style.css.

File: static/css/style.css [hunk 4]
```diff
@@ -132,16 +139,16 @@ strong{
 }
 
 #frontpage{
-	height: 300px;
-	padding: 5px 5px 20px 5px;
+	height: 350px;
+	background: url(groupmeet.jpg) no-repeat fixed;
```
Comment (livike) [inline static/css/style.css]: You are referencing the image in your css folder.
To reference the right image (resize it first), use the right path by adding "../" before the image name: url(../groupmeet.jpg)
A quick tip about paths: http://css-tricks.com/quick-reminder-about-file-paths/

**Pull request merged**